integration-tests/tests/workspace-dashboard.js

```diff
@@ -213,8 +213,11 @@ const testAzureWorkspace = withUserToken(async ({ page, token, testUrl }) => {
   // Verify tabs that currently depend on Google project ID are not present.
   await dashboard.assertTabs(['notebooks', 'workflows', 'job history'], false)
```

Comment:
It would be good to check here that the data tab is NOT present in "prod".

Reply:
As an alternative the may be simpler, I added this line:
```
await dashboard.assertTabs(['data'], environment === 'dev')
```